src/test/fuzz/multiplication_overflow.cpp

```diff
@@ -14,7 +14,7 @@
 #if __has_builtin(__builtin_mul_overflow)
 #define HAVE_BUILTIN_MUL_OVERFLOW
 #endif
-#elif defined(__GNUC__) && (__GNUC__ >= 5)
+#elif defined(__GNUC__)
```

Comment:
isn't this covered by the previous check already?

https://gcc.gnu.org/onlinedocs/gcc-7.1.0/gcc/Integer-Overflow-Builtins.html

Reply:
Thanks! Updated.

Reply:
Reverted back due to @fanquake's [comment](https://github.com/bitcoin/bitcoin/pull/20491#issuecomment-734135165).